__tests__/app.test.js

```diff
@@ -107,6 +107,61 @@ describe("/api/articles/:article_id", () => {
     });
 })
 
+describe("/api/articles/:article_id/comments", () => {
+    test("GET 200: Should return an array of comment objects with the correct keys", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body
+            expect(comments.length).toBe(11)
+            comments.forEach(comment => {
+                expect(typeof comment.comment_id).toBe("number")
+                expect(typeof comment.votes).toBe("number")
+                expect(typeof comment.created_at).toBe("string")
+                expect(typeof comment.author).toBe("string")
+                expect(typeof comment.body).toBe("string")
+                expect(typeof comment.article_id).toBe("number")
+            });
+        })
+    })
+    test("GET 200: Should return an array of comment objects ordered by most recent first", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body
```

Comment:
yaaay lovely split out tests :) 
We dont need to check the length here as there is no forEach